src/components/ZclAttributeManager.vue

```diff
@@ -276,13 +279,33 @@ export default {
         return false
       }
     },
-    isDisabled(id, name, selectedClusterId, selectedCluster) {
+    isDisabledDefault(id, name, selectedClusterId) {
+      return (
+        !this.selection.includes(
+          this.hashAttributeIdClusterId(id, selectedClusterId)
+        ) ||
+        this.checkForcedExternal(name) ||
+        this.selectionStorageOption[
+          this.hashAttributeIdClusterId(id, selectedClusterId)
+        ] == 'External'
+      )
+    },
+    isDisabled(id, name, selectedClusterId) {
```

Comment:
Can these functions have more description as comments? I find it difficult to understand the difference by taking quick look.

Reply:
sure, will do